### ISSUE TITLE
Markdown 3.0.0

### ISSUE THREAD

**Issue by davesvnbrger** [asjqkkkk/markdown_widget#48]
**Describe the bug**
Incompatible with Markdown 3.0.0.  Can't be used if flutter_markdown 0.5.0 is used as well.

```
Because markdown_widget >=1.2.4 depends on markdown ^2.1.8 and flutter_markdown 0.5.0 depends on markdown ^3.0.0, markdown_widget >=1.2.4 is incompatible with flutter_markdown 0.5.0.

And because no versions of flutter_markdown match >0.5.0 <0.6.0, markdown_widget >=1.2.4 is incompatible with flutter_markdown ^0.5.0.

```

**Comment by asjqkkkk** [comment]
Ok, I'll update `markdown`'s version in Saturday😄

**Comment by asjqkkkk** [comment]
Hi, this issue has been solved in v1.2.5😄
